generators/client/templates/angular/tslint.json.ejs

```diff
@@ -117,7 +117,13 @@
         "import-spacing": true,
         "no-consecutive-blank-lines": [true],
         "object-literal-shorthand": true,
-        "space-before-function-paren": [true, "never"],
+        "space-before-function-paren": [true, {
+            "asyncArrow": "always",
+            "anonymous": "never",
+            "constructor": "never",
+            "method": "never",
+            "named": "never"
+        }],
 
```

Comment:
@deepu105 I have modified the tslint config in order to have

Reply:
yup its fine